modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -30,6 +30,7 @@ You can design your API using either RAML or OAS 2.0 and 3.0 specifications in A
 
 Studio leverages APIkit to scaffold the flow and make it easier for you to start developing a Mule application from an API specification.
 
+NOTE: Studio does not support scaffolding API specs that reference OAS or JSON schema fragments imported from Exchange as project dependencies. Instead, you can scaffold a spec with such fragments by providing the fragments inline within the spec or through fragment files that you create manually within your project directory. RAML fragments from Exchange are not affected by this limitation.
```

Comment:
```suggestion
NOTE: Studio does not support scaffolding API specs that reference OAS or JSON schema fragments imported from Exchange as project dependencies. Instead, you can scaffold a spec with these fragments by providing the fragments inline within the spec or through fragment files that you create manually within your project directory. RAML fragments from Exchange are not affected by this limitation.
```

Reply:
The "through" isn't parallel to the "by providing the fragments" so it is awkward.

Maybe more clear:
...or through fragment files that you create manually within your project directory. > or by providing fragment files that you create manually within your project directory.

Reply:
OK! Thanks!